### ISSUE TITLE
Pasta network with rootless Traefik container fails after period of time

### ISSUE THREAD

**Issue by krair** [containers/podman#20032]
### Issue Description

Running Traefik reverse proxy using the pasta network mode will fail after some use. Sometimes after a couple of days, sometimes as quickly as a few minutes. The only thing I can find in the journal is:

```
passt.avx2[1058745]:  Failed to open tun socket in namespace
```

The error seems to occur most frequently when I'm streaming media from a Jellyfin container. Restarting the `systemd` unit for Traefik usually fixes the issue for another period of time. I did not have this issue with the slirp4netns network mode. I get no error logs at all within Traefik itself, but every web service becomes unreachable.

### Steps to reproduce the issue

Steps to reproduce the issue
1. Run Traefik rootless via podman-compose:
```
services:
  traefik:
    image: docker.io/library/traefik:2
    container_name: traefik
    network_mode: "pasta"
    ports:
      - 80:80
      - 443:443
```
2. Use Traefik's "file provider" method (not via exposing the podman socket) to connect to Jellyfin (bound to an internal dummy loopback).

3. Stream media from Jellyfin (music or movies, doesn't seem to matter).

### Describe the results you received

All containers behind the reverse proxy are unreachable (although still reachable via their IP address).

### Describe the results you expected

Traefik to not crash all web services.

### podman info output

```yaml
host:
  arch: amd64
  buildahVersion: 1.29.0
  cgroupControllers:
  - memory
  - pids
  cgroupManager: systemd
  cgroupVersion: v2
  conmon:
    package: conmon-2.1.7-1.el9_2.x86_64
    path: /usr/bin/conmon
    version: 'conmon version 2.1.7, commit: fab2fef7227d2dc16478d29f1185953f81451702'
  cpuUtilization:
    idlePercent: 88.55
    systemPercent: 2.58
    userPercent: 8.87
  cpus: 4
  distribution:
    distribution: '"almalinux"'
    version: "9.2"
  eventLogger: file
  hostname: abcdef
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 1001
      size: 1
    - container_id: 1
      host_id: 165536
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 1001
      size: 1
    - container_id: 1
      host_id: 165536
      size: 65536
  kernel: 5.14.0-284.25.1.el9_2.x86_64
  linkmode: dynamic
  logDriver: k8s-file
  memFree: 2693349376
  memTotal: 32538284032
  networkBackend: netavark
  ociRuntime:
    name: crun
    package: crun-1.8.4-1.el9_2.x86_64
    path: /usr/bin/crun
    version: |-
      crun version 1.8.4
      commit: 5a8fa99a5e41facba2eda4af12fa26313918805b
      rundir: /run/user/1001/crun
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +CRIU +YAJL
  os: linux
  remoteSocket:
    path: /run/user/1001/podman/podman.sock
  security:
    apparmorEnabled: false
    capabilities: CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FOWNER,CAP_FSETID,CAP_KILL,CAP_NET_BIND_SERVICE,CAP_SETFCAP,CAP_SETGID,CAP_SETPCAP,CAP_SETUID,CAP_SYS_CHROOT
    rootless: true
    seccompEnabled: true
    seccompProfilePath: /usr/share/containers/seccomp.json
    selinuxEnabled: true
  serviceIsRemote: false
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: slirp4netns-1.2.0-3.el9.x86_64
    version: |-
      slirp4netns version 1.2.0
      commit: 656041d45cfca7a4176f6b7eed9e4fe6c11e8383
      libslirp: 4.4.0
      SLIRP_CONFIG_VERSION_MAX: 3
      libseccomp: 2.5.2
  swapFree: 14648864768
  swapTotal: 16332615680
  uptime: 331h 19m 2.00s (Approximately 13.79 days)
plugins:
  authorization: null
  log:
  - k8s-file
  - none
  - passthrough
  - journald
  network:
  - bridge
  - macvlan
  volume:
  - local
registries:
  search:
  - registry.access.redhat.com
  - registry.redhat.io
  - docker.io
store:
  configFile: /home/artwo/.config/containers/storage.conf
  containerStore:
    number: 36
    paused: 0
    running: 36
    stopped: 0
  graphDriverName: overlay
  graphOptions: {}
  graphRoot: /home/artwo/.local/share/containers/storage
  graphRootAllocated: 981688012800
  graphRootUsed: 186450092032
  graphStatus:
    Backing Filesystem: xfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
    Using metacopy: "false"
  imageCopyTmpDir: /var/tmp
  imageStore:
    number: 166
  runRoot: /run/user/1001/containers
  transientStore: false
  volumePath: /home/artwo/.local/share/containers/storage/volumes
version:
  APIVersion: 4.4.1
  Built: 1694517849
  BuiltTime: Tue Sep 12 13:24:09 2023
  GitCommit: ""
  GoVersion: go1.19.10
  Os: linux
  OsArch: linux/amd64
  Version: 4.4.1
```

### Podman in a container

No

### Privileged Or Rootless

Rootless

### Upstream Latest Release

No

### Additional environment details

Additional environment details

### Additional information

Additional information like issue happens only occasionally or issue happens with a particular architecture or on a particular setting

**Comment by Luap99** [comment]
I suggest you first try to update podman and pasta to the latest version, at the very least the latest pasta should include a much better error for that condition (https://passt.top/passt/commit/?id=7bc9b66fc2fae6b81ce497a34a5692f63691c810)

`Failed to open tun socket in namespace`, as best I can tell this can only happen on startup and should not happen after some time so that would be weird.
It would also help if you could define what fails mean? Is the process gone, does the interface still exists in the container namespace, are the ip address still there, etc...

**Comment by krair** [comment]
Thank you for the reply.

I apologize for not being more specific. It's generally been crashing during moments when I don't have the time to investigate further. By fail I just meant that no services could be resolved through the reverse proxy. No gateway timeouts, nothing. Which leads me to believe it *could* have been a problem with the network interface.

The container's process still existed, and I was able to interact with it via `podman exec` and look at the logs via `podman logs`, but unable to access anything through the networked ports.

I have installed the latest (https://copr.fedorainfracloud.org/coprs/sbrivio/passt/build/6387922/) build of pasta, and will report back after testing. Previously was the most updated available in the repo (https://almalinux.pkgs.org/9/almalinux-appstream-x86_64/passt-0%5E20230222.g4ddbcb9-4.el9_2.x86_64.rpm.html).
